expan/core/experiment.py

```diff
@@ -37,17 +34,19 @@ def __init__(self, data, metadata):
         }
 
     def __str__(self):
-        return 'Experiment "{:s}" with {:d} entities.'.format(self.metadata['experiment'], len(self.data))
+        return 'Performing "{:s}" experiment.'.format(self.metadata['experiment'])
 
 
-    def analyze_statistical_test(self, test, test_method='fixed_horizon', **worker_args):
+    def analyze_statistical_test(self, test, test_method='fixed_horizon', include_data=False, **worker_args):
```

Comment:
I would rename ```include_data``` to ```exclude_data_in_result```, and in the end of this method you can just write:

```if exclude_data_in_result: del test.data```

Reply:
done!

Reply:
This logic appears strange to me, to be honest. Why would `analyze_statistical_test` want to touch the inner fields of `test`?